Project2015To2017/Definition/AssemblyAttributes.cs

```diff
@@ -1,16 +1,174 @@
+using System;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
 namespace Project2015To2017.Definition
 {
-    public sealed class AssemblyAttributes
-    {
-	    public string Title { get; set; }
-        public string Company { get; set; }
-        public string Product { get; set; }
-        public string Copyright { get; set; }
-        public string InformationalVersion { get; set; }
-        public string Version { get; set; }
-        public string AssemblyName { get; set; }
-	    public string Description { get; set; }
-	    public string Configuration { get; set; }
-	    public string FileVersion { get; set; }
-    }
+	public sealed class AssemblyAttributes
+	{
+		public CompilationUnitSyntax FileContents { get; set; }
```

Comment:
This is not a place to use Roslyn in my opinion. We're not doing anything fancy here so this is just overhead and an additional dependency that we don't need to bring in.

Reply:
Actually, I see how you're using this now on the read side. We could use it there but then I would argue for loading the entire project to begin with 😄 I'll make a mental note of that as an improvement down the road. Ignore the above comment